WalletWasabi.Documentation/BackendDeployment.md

```diff
@@ -15,7 +15,7 @@ bitcoind
 bitcoin-cli getblockchaininfo
 tor
 sudo service nginx start
-dotnet publish ~/WalletWasabi/WalletWasabi.Backend --configuration Release --self-contained false
+rm WalletWasabi/WalletWasabi.Backend/bin/ -r ; dotnet publish ~/WalletWasabi/WalletWasabi.Backend --configuration Release --self-contained false
```

Comment:
```bash
rm -rf WalletWasabi/WalletWasabi.Backend/bin &&  dotnet publish ~/WalletWasabi/WalletWasabi.Backend --configuration Release --self-contained false
```

* `rm -rf <directory>` is the way. It is one of few commands that every bash user remember.
* `command1 && command2` is more common because it means that `command2` has to be executed **only** if `command1` was successful (I think this is what we want)

Reply:
I was curious if there isn't any better way and I found the following idea in internet: add a new task in the `WalletWasabi.Backend.csproj` to delete the `/bin` directory after the command `dotnet clean`.

```json
<Target Name="AfterClean">
    <Exec Command="rm -rf .\bin" />
</Target>
``` 
I don't know if it worth the effort but looks like a nice solution.

Reply:
This is not working on Windows

Reply:
btw: PowerShell is multi-plaform for some time. That may be an option too :-) (expecting a few blows :D)